server/app.ts

```diff
@@ -77,7 +77,8 @@ export default function createApp(services: Services): express.Application {
     }),
   )
   app.use(breadcrumbs())
-  app.use(checkPopulateUserCaseloads(services.prisonApiService, services.csipApiService))
+  app.use(dpsComponents.retrieveCaseLoadData({ logger }))
+  app.use(checkServiceEnabledForActiveCaseLoad(services.csipApiService))
```

Comment:
This app had some bespoke logic around checking that the active case load was one that the service supported.  I've separated that out into it's own middleware here.